featurebyte/query_graph/sql/groupby_helper.py

```diff
@@ -23,9 +23,17 @@ class GroupbyColumn:
     """
 
     agg_func: AggFunc
+    # parent_expr refers to the expression that is used to generate the column. For example, `sum(col1)`.
+    # This is not provided for certain aggregations like count.
     parent_expr: Optional[Expression]
+    # parent_dtype is the dtype of the parent column.
     parent_dtype: Optional[DBVarType]
+    # result_name is the aliased name of the aggregation result. For example if we want `sum(col1) as "sum_col_1",
+    # the result_name is "sum_col_1".
     result_name: str
+    # parent_cols refers to the columns used in the parent_expr. As an example, for the aggregation `sum(col1)`, the
+    # parent_cols here will be ['col1'].
+    parent_cols: List[Expression]
```

Comment:
we add this in here as it'll be used by some of the vector aggregation operations.